src/store/channels/api.ts

```diff
@@ -21,3 +21,14 @@ export async function markAllMessagesAsReadInChannel(channelId: string, userId:
 
   return response.status;
 }
+
+export type MentionableUser = {
+  id: string;
+  name: string;
+};
+
+export async function searchMentionableUsers(channelId: string, search: string): Promise<MentionableUser[]> {
+  return await get<any>(`/chatChannels/${channelId}/mentionable-users`, search)
```

Comment:
I think
```suggestion
  return await get<MentionableUser[]>(`/chatChannels/${channelId}/mentionable-users`, search)
```

Reply:
Hmm.. I can't actually figure out where this impacts anything but, I agree, it's probably correct.